### ISSUE TITLE
[5.12.4] Dashboard Knowledge graph queries are broken

### ISSUE THREAD

**Issue by MaxwellDPS** [LuatixHQ/opencti#5400]
## Description

When making a dashboard, if you pick any visualization type aside list then use a relations filter it will show the red  An unknown error occurred. Please contact your administrator or the OpenCTI maintainers." error banner on the widget when saved.

TLDR; relations filters break any non-list widget

## Environment

1. OS (where OpenCTI server runs): CENTOS Stream 9
2. OpenCTI version: 5.12.15
3. OpenCTI client: Platform
4. Other environment details: Clustered on k8s

## Reproducible Steps

Steps to create the smallest reproducible scenario:
1. Create a Dashboard
2. add a widget
3. select number or line graph for example
4. select "Knowledge graph" for the query type
5. Add a relationship type and an from entity type
6. Save 
7. Dashboard loads and graph breaks

## Expected Output

Working dashboard filters with verbose output of the error

## Actual Output
A broken widget that displays a red banner with
"An unknown error occurred. Please contact your administrator or the OpenCTI maintainers."

#### Chrome Console
```
front-L5XQQ6M2.js:8 TypeError: x.filter is not a function
    at _ (front-L5XQQ6M2.js:42065:547)
    at duu (front-L5XQQ6M2.js:42065:2675)
    at lit (front-L5XQQ6M2.js:6:19878)
    at Kka (front-L5XQQ6M2.js:8:44787)
    at Ika (front-L5XQQ6M2.js:8:40415)
    at I7s (front-L5XQQ6M2.js:8:40338)
    at Man (front-L5XQQ6M2.js:8:40185)
    at Rat (front-L5XQQ6M2.js:8:36454)
    at Tka (front-L5XQQ6M2.js:8:35374)
    at ytt (front-L5XQQ6M2.js:1:10975)
vat @ front-L5XQQ6M2.js:8
4front-L5XQQ6M2.js:8 Error
    at Object.render (front-L5XQQ6M2.js:8441:46772)
    at a.render (front-L5XQQ6M2.js:40:46528)
    at _at (front-L5XQQ6M2.js:8:6275)
    at Aba (front-L5XQQ6M2.js:8:6067)
    at Kka (front-L5XQQ6M2.js:8:45661)
    at Ika (front-L5XQQ6M2.js:8:40415)
    at I7s (front-L5XQQ6M2.js:8:40338)
    at Man (front-L5XQQ6M2.js:8:40185)
    at Rat (front-L5XQQ6M2.js:8:36454)
    at Tka (front-L5XQQ6M2.js:8:35374)
vat @ front-L5XQQ6M2.js:8
front-L5XQQ6M2.js:8 Error
    at Object.render (front-L5XQQ6M2.js:8441:46772)
    at a.render (front-L5XQQ6M2.js:40:46528)
    at _at (front-L5XQQ6M2.js:8:6275)
    at Aba (front-L5XQQ6M2.js:8:6067)
    at Kka (front-L5XQQ6M2.js:8:45661)
    at Ika (front-L5XQQ6M2.js:8:40415)
    at I7s (front-L5XQQ6M2.js:8:40338)
    at Man (front-L5XQQ6M2.js:8:40185)
    at Rat (front-L5XQQ6M2.js:8:36454)
    at Tka (front-L5XQQ6M2.js:8:35374)
vat @ front-L5XQQ6M2.js:8
o.componentDidCatch.a.callback @ front-L5XQQ6M2.js:8
gba @ front-L5XQQ6M2.js:6
wba @ front-L5XQQ6M2.js:8
Aka @ front-L5XQQ6M2.js:8
Aka @ front-L5XQQ6M2.js:8
Aka @ front-L5XQQ6M2.js:8
T7s @ front-L5XQQ6M2.js:8
w7s @ front-L5XQQ6M2.js:8
Ade @ front-L5XQQ6M2.js:8
Tka @ front-L5XQQ6M2.js:8
ytt @ front-L5XQQ6M2.js:1
ctt @ front-L5XQQ6M2.js:1
````

#### API Response
```
{
    "errors": [
        {
            "message": "parsing_exception\n\tCaused by:\n\t\tx_content_parse_exception: [1:896] [bool] failed to parse field [must]\n\tRoot causes:\n\t\tparsing_exception: Failed to parse",
            "locations": [
                {
                    "line": 21,
                    "column": 5
                }
            ],
            "path": [
                "stixRelationshipsNumber",
                "total"
            ],
            "extensions": {
                "code": "INTERNAL_SERVER_ERROR"
            }
        }
    ],
    "data": {
        "stixRelationshipsNumber": null
    }
}
```

## Additional information

example dashboard that is having this issue

```
{
    "openCTI_version": "5.12.4",
    "type": "dashboard",
    "configuration": {
        "name": "Example",
        "manifest": "eyJ3aWRnZXRzIjp7IjRkZmE3N2EzLWIwMjMtNDIwZi05NDBiLTQyNmNlMWUyNjgwZSI6eyJpZCI6IjRkZmE3N2EzLWIwMjMtNDIwZi05NDBiLTQyNmNlMWUyNjgwZSIsInR5cGUiOiJudW1iZXIiLCJwZXJzcGVjdGl2ZSI6InJlbGF0aW9uc2hpcHMiLCJkYXRhU2VsZWN0aW9uIjpbeyJsYWJlbCI6IiIsImF0dHJpYnV0ZSI6ImVudGl0eV90eXBlIiwiZGF0ZV9hdHRyaWJ1dGUiOiJjcmVhdGVkX2F0IiwicGVyc3BlY3RpdmUiOiJyZWxhdGlvbnNoaXBzIiwiaXNUbyI6dHJ1ZSwiZmlsdGVycyI6eyJtb2RlIjoiYW5kIiwiZmlsdGVycyI6W3sia2V5IjpbInJlbGF0aW9uc2hpcF90eXBlIl0sInZhbHVlcyI6WyJ0YXJnZXRzIl0sIm9wZXJhdG9yIjoiZXEiLCJtb2RlIjoib3IifSx7ImtleSI6WyJmcm9tVHlwZXMiXSwidmFsdWVzIjpbIkludHJ1c2lvbi1TZXQiXSwib3BlcmF0b3IiOiJlcSIsIm1vZGUiOiJvciJ9XSwiZmlsdGVyR3JvdXBzIjpbXX0sImR5bmFtaWNGcm9tIjp7Im1vZGUiOiJhbmQiLCJmaWx0ZXJzIjpbXSwiZmlsdGVyR3JvdXBzIjpbXX0sImR5bmFtaWNUbyI6eyJtb2RlIjoiYW5kIiwiZmlsdGVycyI6W10sImZpbHRlckdyb3VwcyI6W119fV0sInBhcmFtZXRlcnMiOnt9LCJsYXlvdXQiOnsidyI6NSwiaCI6MywieCI6MSwieSI6MCwiaSI6IjRkZmE3N2EzLWIwMjMtNDIwZi05NDBiLTQyNmNlMWUyNjgwZSIsIm1vdmVkIjpmYWxzZSwic3RhdGljIjpmYWxzZX19fSwiY29uZmlnIjp7fX0="
    }
}
```

**Comment by Archidoit** [comment]
I can't reproduce your error with these filters: 
![image](https://github.com/OpenCTI-Platform/opencti/assets/75783086/83e7959a-ee2d-458a-8255-a2de29d5b1e6)

Do you still have this issue with the last version of openCTI? (5.12.15)

**Comment by Archidoit** [comment]
Hello! I can't reproduce your error with these filters: 
![image](https://github.com/OpenCTI-Platform/opencti/assets/75783086/f2d50dd6-e7a0-4d04-a8eb-da907dda2ad1)

Do you still have an error with the last version of OpenCTI (5.12.15) ?

**Comment by SamuelHassine** [comment]
Bug known and solved in the most recent 5.12.X releases.
